ReferenceCore/ExtenderDoc.cs

```diff
@@ -174,26 +246,33 @@ public ExtenderDoc(IDocRenderer renderer)
             tableStringBuilder.Append(_renderer.RenderNewParagraph());
 
             var dict = new Dictionary<string, string>();
-            var regex = new Regex("[^a-zA-Z0-9 -]");
+
 
             foreach(var member in members) {
                 var memberNameTransformed = memberNameTransform == null ? member.Name : memberNameTransform(member);
-                var memberNameLink = "#" + regex.Replace(memberNameTransformed.ToLower(), "").Replace(" ", "-");
+                string memberAnchor = GenerateAnchor(memberNameTransformed);
 
                 var remarks = "";
                 if(renderRemarks
                     &&
                     !String.IsNullOrWhiteSpace(member.Remarks))
                     remarks = _renderer.RenderText("Remarks:", italic: true, bold: true) + " " + _renderer.RenderText(_renderer.Sanitize(member.Remarks).Trim(), italic: true);
 
-                dict.Add(generateMemberLink ? String.Format("[{0}]({1})", memberNameTransformed, memberNameLink) : memberNameTransformed,
+                dict.Add(generateMemberLink
+                        ? _renderer.RenderUrl(memberNameTransformed, memberAnchor)
+                        : memberNameTransformed,
                     _renderer.Sanitize(member.Summary) + _renderer.RenderLineBreak() + remarks);
             }
             tableStringBuilder.Append(_renderer.RenderDescriptionBlock(dict));
 
             _markupStringBuilder.Append(tableStringBuilder.ToString());
         }
 
+        private static string GenerateAnchor(string text) {
```

Comment:
No. It will replace  `'_'` with the `'-'` character, which is wrong.
Also, `Trim('-')` won't replace a character in the middle of a string.